RadixWallet/Features/SettingsFeature/Preferences/Preferences+View.swift

```diff
@@ -0,0 +1,182 @@
+private typealias S = L10n.Preferences
```

Comment:
I don't like this, though I think Alex does. Since you don't seem to be using these strings in single row contexts anyway, there is no space constraint, better to use the full version, saves some cognitive load.

Reply:
maybe we can put it up to a vote then? cc @CyonAlexRDX @GhenadieVP @danvleju-rdx 
I see two benefits of this approach:
- Save saves spaces and makes usage of localization shorter
- Prevents from adding the wrong string by mistake. For example, it is easy to confuse and autocomplete with:
```Swift
.navigationTitle(L10n.AppSettings.title)
```
when you actually needed:
```Swift
.navigationTitle(L10n.AccountSettings.title)
```

A similar case like this happened to me while working on this PR.

Reply:
The downside is that it makes it hard to read, you have to scroll up to see what this means. In this case, there is no reason to use it since space is not at a premium.

For the second argument, that is good as far as it goes, but the relation between screens and string groups is not entirely 1-to-1.

But the biggest downside is that it makes some searches impossible. I often search for parts of the string key.

Reply:
Actually the biggest reason I am against this is that it makes strings less visually distinctive, so if you guys really insist on this I think we should use something better than `S`. Perhaps `L13g` for "localisedstring" or something clever.

Reply:
Yes that's why I wrote "some" searches. But even here, it gives you the file, not the line, so you can't tell from the left pane search results, you need to go into the file.

Reply:
Since it's not needed for this screen - there is no space constraint, let's postpone this and use the full key for now?